src/planning/plan/planificator.ts

```diff
@@ -151,15 +151,17 @@ export class Planificator {
   private _listenToArcStores() {
     this.arc.onDataChange(this.dataChangeCallback, this);
     this.storeCallbackIds = new Map();
-    this.arc.context.allStores.forEach(async store => {
+    this.arc.context.allStores.forEach(async storeInfo => {
+      const store = this.arc.context.getActiveStore(storeInfo);
```

Comment:
context shouldn't be providing active stores; this goes back to the idea that storeInfo (which was previously just store) is the behavior-free representation of a store, but the active store needs to be created in the context of a running arc.

Reply:
removed ActiveStore and StorageService from manifest